crates/bevy_ui/src/flex/mod.rs

```diff
@@ -285,14 +285,17 @@ pub fn flex_node_system(
         if node.size != new_size {
             node.size = new_size;
         }
-        let position = &mut transform.translation;
-        position.x = to_logical(layout.location.x + layout.size.width / 2.0);
-        position.y = to_logical(layout.location.y + layout.size.height / 2.0);
+        let mut new_position = transform.translation;
```

Comment:
Please leave a comment as to why this is the case so that this doesn't regress in the future.

Reply:
I'll add a comment, but this seems like a pattern that we should use everywhere that we're not 100% sure the value will not be the same. Otherwise we're potentially doing a lot of extra work and breaking `Changed` guarantees. This shouldn't cause bugs, but it's something that can affect performances negatively.